http/src/client/builder.rs

```diff
@@ -117,6 +122,12 @@ impl ClientBuilder {
 
         self
     }
+    /// Set a group headers which are sent in every request.
```

Comment:
```suggestion

    /// Set a group headers which are sent in every request.
```
Please add a newline here